Dockerfile

```diff
@@ -108,12 +108,15 @@ WORKDIR /app
 COPY --chown=appuser:appuser  . .
 COPY --from=deps --chown=appuser:appuser /workspace-install ./
 
-# Optional: if the app depends on global /static shared assets like images, locales...
-
 RUN yarn install --immutable --inline-builds
 RUN yarn workspace ${PROJECT} share-static-hardlink
 RUN yarn workspace ${PROJECT} build
 
+# Hack to patch the bug in Next.js standalone builds with monorepos:
```

Comment:
Can you add the link to the issue so it's easier to follow if there will be a resolution later?